device-types/HPE/ProLiant-DX385-Gen10-Plus.yaml

```diff
@@ -0,0 +1,58 @@
+---
+manufacturer: HPE
+model: ProLiant DX385 Gen10 Plus
+slug: hpe-proliant-dx385-gen10-plus
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+front_image: true
+rear_image: true
+comments: '[ProLiant DX385 Gen10 Plus Server Specs](https://www.hpe.com/psnow/doc/a00056112enw)'
+console-ports:
+  - name: VGA
+    type: other
```

Comment:
VGA is not a console port type. Please remove this

Reply:
fixed